docs/metrics.md

```diff
@@ -12,6 +12,9 @@ All metrics documented here are auto-generated by the utility tool `tools/doc-ge
 ### kubevirt_info
 Version information.
 
+### kubevirt_configuration_emulation_enabled
```

Comment:
Please add a note that says that is this is false that means that no VM can run

Reply:
[emulation configuration](https://github.com/kubevirt/kubevirt/blob/main/docs/software-emulation.md) is just an alternative to the preferred kvm hardware emulation, so it does not mean per se that no VM can run